packages/notepadplusplus.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,32 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  # Path to Notepad++'s configuration file
+  $configFile = Join-Path ${Env:APPDATA} "Notepad++\config.xml"
+
+  if (-Not (Test-Path -Path $configFile)) {
+    # Launch Notepad++
+    Start-Process "notepad++"
+    Start-Sleep -Seconds 2
+
+    # Try to gracefully close it so that it can create it's config.xml
+    # Give it a little time in case it's slow
+    foreach ($item in "0", "1") {
+      $notepad = Get-Process -Name "notepad++" -ErrorAction SilentlyContinue
+      if ($notepad -ne $null) {
+        $notepad | ForEach-Object {$_.CloseMainWindow() | Out-Null}
+      }
+      Start-Sleep -Seconds 2
+    }
+  }
+
+  if (-Not (Test-Path -Path $configFile)) {
+    Write-Host "Can't find config.xml to modify"
```

Comment:
Instead of `Write-Host`, use `VM-Write-Log "ERROR" "Can't find config.xml to modify"`

Reply:
I can use `Write-Host` 👍  Although I think this is more a `WARN` than a `ERROR`, notepad is still installed, just the updates are not disabled. 🤔